### ISSUE TITLE
Ta betalt

### ISSUE THREAD

**Issue by d-tamm** [d-tamm/ff-boka#8]
Hade varit kul att kunna ta betalt direkt i bokningssystemet. 
* Swish https://www.swish.nu/use-cases/swish-online (kräver Swish för handel; går det att koppla till de olika lokalavdelningarna?)
* Kreditkort (via Paypal?)

**Comment by d-tamm** [comment]
Inte aktuellt för första versionen.

**Comment by carlcrafoord** [comment]
I webben har vi möjlighet att ta betalt för aktiviteten, det kan vara ett sätt att få betalt. Antingen att aktiviteten har två priser tex på grupp i aktiviten. tex en kajak verksamhet som har ett pris för de med egen kajak och ett annat pris för de som vill hyra.

**Comment by d-tamm** [comment]
Ja, men då pratar vi äventyr. Där behöver vi inte uppfinna hjulet igen. Det som den här issue gäller är att ta betalt vid uthyrning av resurser, såsom stugor, kanoter mm, när det inte finns något tillhörande äventyr.

**Comment by d-tamm** [comment]
För Swish måste  kassör i varje LA ladda ner sina egna certifikat och sedan installera den i bokningen på LA-nivå

**Comment by d-tamm** [comment]
Kan finnas önskemål om att på något sätt integrera med äventyrshanteraren där folk också ska betala ibland.
